features/car/traffic_turn_penalties.feature

```diff
@@ -62,24 +60,24 @@ Feature: Traffic - turn penalties
             """
         And the contract extra arguments "--turn-penalty-file penalties.csv"
         When I route I should get
-            | from | to | route                    | speed   | time      |
-            | a    | h  | ad,dhk,dhk               | 63 km/h | 11.5s +-1 |
+            | from | to | route                 | speed   | time      |
+            | a    | h  | ad,dhk,dhk            | 63 km/h | 11.5s +-1 |
                                                                               # straight
-            | i    | g  | fim,fg,fg                | 55 km/h | 13s +-1   |
+            | i    | g  | fim,fg,fg             | 55 km/h | 13s +-1   |
                                                                               # right - ifg penalty
-            | a    | e  | ad,de,de                 | 64 km/h | 11s +-1   |
+            | a    | e  | ad,def,def            | 64 km/h | 11s +-1   |
                                                                               # left - faster because of negative ade penalty
-            | c    | g  | cd,de,ef,fg,fg           | 63 km/h | 23s +-1   |
+            | c    | g  | cd,def,fg,fg          | 63 km/h | 23s +-1   |
                                                                               # double straight
-            | p    | g  | mp,fim,fg,fg             | 59 km/h | 24.5s +-1 |
+            | p    | g  | mp,fim,fg,fg          | 59 km/h | 24.5s +-1 |
                                                                               # straight-right - ifg penalty
-            | a    | l  | ad,de,ef,fim,lm,lm       | 61 km/h | 35.5s +-1 |
+            | a    | l  | ad,def,fim,klm,klm    | 61 km/h | 35.5s +-1 |
                                                                               # was straight-left - forced around by hkl penalty
-            | l    | e  | lm,fim,ef,ef             | 57 km/h | 25s +-1   |
+            | l    | e  | klm,fim,def,def       | 57 km/h | 25s +-1   |
                                                                               # double right - forced left by lkh penalty
-            | g    | n  | fg,fim,mn,mn             | 30 km/h | 47.5s +-1   |
+            | g    | n  | fg,fim,mn,mn          | 30 km/h | 47.5s +-1   |
```

Comment:
There is a cool Vim plugin called [vim-easy-align](https://github.com/junegunn/vim-easy-align) (helps not only for cucumber but in general for aligning).
It can be used e.g. as in

    gaip*|

as in
- `ga` start easy align
- `ip` in paragraph (the whole cucumber section)
- `*|` around `|`s

/cc @emiltin @TheMarex @danpat